### PR TITLE
{bio}[foss/2021b] MIGRATE-N v5.0.4

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/3e9734be7e4c624154ef7a575a406a74 for a full test report.

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/637363d69936ea87505758db00412e60 for a full test report.

File: easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-5.0.4-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    'migrate-%(version)s_locale.patch',
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+]
+
+configopts = '--enable-A4 "CC=$CC" STDCPLUS=-lstdc++'
+prebuildopts = 'make mpis && make clean &&'
+installopts = "&& make clean && make installmpi"
```
Comment (boegel) [inline easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-5.0.4-foss-2021b.eb:30]: The `make clean` commands are actually not needed at all, the installation works fine without them

File: easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-5.0.4-foss-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+parallel = 1
+
+start_dir = 'src'
```
Comment (boegel) [inline easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-5.0.4-foss-2021b.eb:34]: Please move both `start_dir` and `parallel` up, above `configopts`.

The (informal) policy is to specify easyconfig parameter in the order in which they're used by EasyBuild

File: easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-5.0.4-foss-2021b.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['migrate-n', 'migrate-n-mpi']],
+    'dirs': [],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-5.0.4-foss-2021b.eb:39]: We can also add a sanity check command:

```python
sanity_check_commands = ["echo Q | migrate-n"]
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/4c6b859370c8c40d94d4d3951991c026 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3106.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/3bd4da6fd4f6e13be8e597a50d34ee25 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**